app/javascript/components/StartEndModal/StartEndModal.jsx

```diff
@@ -38,37 +39,33 @@ const StartEndModal = ({ gameState, startEnd, errors }) => {
   const [selectedTeamId, setSelectedTeamId] = useState(first_team_id || '');
 
   const [open, setOpen] = useState(false);
+  const [teams] = useState(teams_with_players.map((team, index) => {
+      return ( <MenuItem key={index} value={team.team.id}>{team.team.team_name}</MenuItem> );
```

Comment:
I don't think it's normal to use state to store an array of rendered components...